graspy/utils/ptr.py

```diff
@@ -3,7 +3,7 @@
 from scipy.stats import rankdata
 
 
-def pass_to_ranks(graph, method='zero-boost', *args):
+def pass_to_ranks(graph, method='zero-boost', **kwargs):
```

Comment:
whats the point of kwargs?

Reply:
wanted to be able to iterate over a list of transformations (eg different forms of PTR) by passing kwargs from a list

Reply:
Something like 

forms = ['zero-boost','simple-nonzero']
for form in forms:
   kwargs = {'method':form}
   graph = pass_to_ranks(graph, **kwargs)